### ISSUE TITLE
JSON cleanup

### ISSUE THREAD

**Issue by vgrichina** [nearprotocol/nearcore#1062]
JSON currently has such problems:
- bigint amounts are returned as hex string without `0x` which makes them both unreadable to humans and hard to parse in universal fashion
- blobs are returned as array of numbers

I think instead of this following should happen:
- bigint amounts should be returned as decimal strings (readable to humans, default base when parsing)
- small blobs (like hashes) can be represented as hex strings with `0x` prefix
- large blobs need to be base64-encoded

**Comment by ilblackdragon** [comment]
For RPC I think we implemented this.
